src/test/java/core/basesyntax/service/impl/ReportGeneratorImplTest.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.service.ReportGenerator;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class ReportGeneratorImplTest {
+    private final Map<Fruit, Integer> testMap = new HashMap<>();
+    private ReportGenerator reportGenerator = new ReportGeneratorImpl();

```

Comment:
You should instantiate ReportGenerator in the @BeforeEach method to ensure it's a fresh instance for each test.

Reply:
ok, done